src/style.css

```diff
@@ -2,0 +1,86 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0 8px;
+
+  --big-box-shadow: 0 4px 4px 0 #00000040;
+  --small-box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.form--big {
+  margin: 20px 0;
+}
+
+.input {
+  width: 100%;
+  background: url('./images/Search.svg') no-repeat;
+  box-sizing: border-box;
+  background-size: contain;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  font-weight: 300;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir-Heavy, sans-serif;
+  color: #3d4e61;
+}
+
+.input::placeholder {
+  font-family: Avenir-Book, sans-serif;
```

Comment:
Consider adding a generic font family like 'sans-serif' as a fallback for 'Avenir-Book' to ensure proper rendering if the custom font fails to load.